tests/headers/objc_class.h

```diff
@@ -1,4 +1,4 @@
-// bindgen-flags: --objc-extern-crate -- -x objective-c
```

Comment:
Is `--objc-extern-crate` still needed under any circumstances ? Since it no longer appears in any expectations, should we be thinking about removing the option, or else about re-adding expectations for it ?

Reply:
I'm up to remove it. I'd only argue that it should be an outside PR/issue to this. The main reason I removed it from all the tests expectations is because all of those expectations are modules and modules can't have `#[macro_use] extern crate objc` as that goes in a top level `src/lib.rs`/`src/main.rs` of a rust crate for rust 2015 edition crate.

Reply:
Though I have not tried to understand the flag itself, I agree that removing it would be out of scope for this PR. As long as we do not forget about the flag's presence, it seems fine not to deal further with it here.